promgen/templates/promgen/farm_detail.html

```diff
@@ -4,7 +4,11 @@
 {% block content %}
 
 <div class="page-header" v-pre>
-  <h1>Farm: {{ farm.name }} ({{ farm.source }})</h1>
+  <h1>Farm: {{ farm.name }} ({{ farm.source }})
+    {% if farm.owner %}
+      <small class="pull-right">{% trans 'Contact' %}: {{farm.owner.username}}</small>
```

Comment:
```suggestion
      <small class="pull-right">{% trans "Contact" %}: {{farm.owner.username}}</small>
```

Use double quotes as much as possible.

Reply:
Fixed in d176e52.